superset/views.py

```diff
@@ -1253,6 +1254,23 @@ def slice(self, slice_id):
         viz_obj = self.get_viz(slice_id)
         return redirect(viz_obj.get_url(**request.args))
 
+    @log_this
+    @has_access_api
+    @expose(
+        "/update_explore/<datasource_type>/<datasource_id>/", methods=['POST'])
+    def update_explore(self, datasource_type, datasource_id):
+        form_data = json.loads(request.form.get('data'))
+        error_redirect = '/slicemodelview/list/'
+        try:
```

Comment:
shouldn't we be using one of these methods here... 

![screenshot 2016-11-11 11 38 10](https://cloud.githubusercontent.com/assets/130878/20227878/6005a2a0-a803-11e6-9061-84f774f439a9.png)